android/.gitignore

```diff
@@ -0,0 +1,5 @@
+/Cuberite/
+/CMakeFiles/
+/lua_native-prefix/
+/tolua_native-prefix/
+CMakeCache.txt
```

Comment:
Just for the record, Linux requirements for newlines at the end of a file is very disagreeable to me.